DF_UDF_README.md

```diff
@@ -15,9 +15,12 @@ commands.
 
 ## Setup
 
-To do this include com.nvidia:df_udf_plugin as a dependency for your project and also include it on the 
-classpath for your Apache Spark environment. Then include `com.nvidia.spark.DFUDFPlugin` in the config 
-`spark.sql.extensions`. Now you can implement a UDF in terms of Dataframe operations.
+Accelerator for Apache Spark. As such you will need to select a scala version 2.12 or 2.13 that matches the
```

Comment:
It feels something is missing from this sentence. It should at least be "RAPIDS Accelerator for Apache Spark", but my guess is there is a copy and paste error.

Reply:
Thanks I totally missed that.